spec/models/links_update_spec.rb

```diff
@@ -77,8 +77,17 @@
         expect(tag_mapping.state).to eql("errored")
       end
 
-      it "concatenates the error messages and sets them on the tag mapping" do
-        expect(tag_mapping.message).to eql("Broken. Rubbish.")
+      it "assigns the error messages to the record" do
+        expect(tag_mapping.messages).to eql(["Broken.", "Rubbish."])
+      end
+
+      it 'does changes the state of the tagging source to errored' do
```

Comment:
Yes, that sentence doesn't make sense. I spotted another one like that, fixing now.